src/main/java/seedu/address/model/person/Module.java

```diff
@@ -22,18 +22,18 @@
     /**
      * Constructs an {@code Address}.
```

Comment:
I think you forgotten to change this to module

Reply:
Noted with thanks!